Hardware/SMBIOS.cs

```diff
@@ -429,6 +433,222 @@ public class ProcessorInformation : Structure {
+        AMDAthlonIIDualCoreM,
+        AMDOpteron6100Series,
+        AMDOpteron4100Series,
+        AMDOpteron6200Series,
+        AMDOpteron4200Series,
+        AMDFXSeries,
+        MIPS,
+        MIPSR4000,
+        MIPSR4200,
+        MIPSR4400,
+        MIPSR4600,
+        MIPSR10000,
+        AMDCSeries,
+        AMDESeries,
+        AMDASeries,
+        AMDGSeries,
+        AMDZSeries,
+        AMDRSeries,
+        AMDOpteron4300Series,
+        AMDOpteron6300Series,
+        AMDOpteron3300Series,
+        AMDFireProSeries,
+        SPARC,
+        SuperSPARC,
+        microSPARCII,
+        microSPARCIIep,
+        UltraSPARC,
+        UltraSPARCII,
+        UltraSPARCIIi,
+        UltraSPARCIII,
+        UltraSPARCIIIi,
+        Motorola68040 = 96,
+        Motorola68xxx,
+        Motorola68000,
+        Motorola68010,
+        Motorola68020,
+        Motorola68030,
+        AMDAthlonX4QuadCore,
+        AMDOpteronX1000Series,
+        AMDOpteronX2000Series,
+        AMDOpteronASeries,
+        AMDOpteronX3000Series,
+        AMDZen,
+        Hobbit = 112,
+        CrusoeTM5000 = 120,
+        CrusoeTM3000,
+        EfficeonTM8000,
+        Weitek = 128,
+        IntelItanium = 130,
+        AMDAthlon64,
+        AMDOpteron,
+        AMDSempron,
+        AMDTurio64Mobile,
+        AMDOpteronDualCore,
+        AMDAthlon64X2DualCore,
+        AMDTurion64X2Mobile,
+        AMDOpteronQuadCore,
+        AMDOpteronThirdGen,
+        AMDPhenomFXQuadCore,
+        AMDPhenomX4QuadCore,
+        AMDPhenomX2DualCore,
+        AMDAthlonX2DualCore,
+        PA_RISC,
+        PA_RISC8500,
+        PA_RISC8000,
+        PA_RISC7300LC,
+        PA_RISC7200,
+        PA_RISC7100LC,
+        PA_RISC7100,
+        V30 = 160,
+        IntelXeon3200QuadCoreSeries,
+        IntelXeon3000DualCoreSeries,
+        IntelXeon5300QuadCoreSeries,
+        IntelXeon5100DualCoreSeries,
+        IntelXeon5000DualCoreSeries,
+        IntelXeonLVDualCore,
+        IntelXeonULVDualCore,
+        IntelXeon7100Series,
+        IntelXeon5400Series,
+        IntelXeonQuadCore,
+        IntelXeon5200DualCoreSeries,
+        IntelXeon7200DualCoreSeries,
+        IntelXeon7300QuadCoreSeries,
+        IntelXeon7400QuadCoreSeries,
+        IntelXeon7400MultiCoreSeries,
+        IntelPentiumIIIXeon,
+        IntelPentiumIIISpeedStep,
+        IntelPentium4,
+        IntelXeon,
+        AS400,
+        IntelXeonMP,
+        AMDAthlonXP,
+        AMDAthlonMP,
+        IntelItanium2,
+        IntelPentiumM,
+        IntelCeleronD,
+        IntelPentiumD,
+        IntelPentiumExtreme,
+        IntelCoreSolo,
+        IntelCore2Duo = 191,
+        IntelCore2Solo,
+        IntelCore2Extreme,
+        IntelCore2Quad,
+        IntelCore2ExtremeMobile,
+        IntelCore2DuoMobile,
+        IntelCore2SoloMobile,
+        IntelCorei7,
+        IntelCeleronDualCore,
+        IBM390,
+        PowerPCG4,
+        PowerPCG5,
+        ESA390G6,
+        zArchitecture,
+        IntelCorei5,

```

Comment:
Core**I**5, I3, I7, I9 and also Intel**I860** and Intel**I960**. The rest looks good to me.

Reply:
The branding on the Core chips is definitely lower case i https://www.intel.com/content/www/us/en/products/processors/core/i7-processors/i7-9700k.html

Ditto i960 https://en.wikipedia.org/wiki/Intel_i960